components/FileFilters.vue

```diff
@@ -102,23 +125,49 @@ const handleConfirm = () => {
       <div v-if="isFilterOpen" class="filters__selection">
         <div class="filter">
           <BaseInput
-            v-model="nameFilter"
+            v-if="storage.bucket === 'private'"
```

Comment:
maybe it would be easier with one BaseInput and condition on `v-model` 🤔 or it is impossible and throws and error?

Reply:
didn't try but maybe it is possible with `computed` with `get` and `set` object inside 🤔 
maybe something like this:
```
const modelName = computed({
  get: () {
    return storage.bucket === 'private' ? privateName : publicName
  },
  set: () {
    something similar
  }
}
```

but don't know if it will work 🤷

Reply:
Condition on v-model 
v-model="storage.bucket === 'private' ? privateFilters.name : publicFilters.name"
results in error "'v-model' directives require the attribute value which is valid as LHS"